utilities/ubi8-bats/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
should we start using the .io registry? needs auth, not sure if that's been thought about when doing a build via gh-actions

Reply:
Are there benefits using `registry.redhat.io` over `registry.access.redhat.com`?

Reply:
`registry.access.redhat.com` will be decommissioned sometime in the coming year. It is recommended that we prepare to use `registry.redhat.io`.  However, that registry does require authentication and supports registry tokens.

https://access.redhat.com/RegistryAuthentication

Reply:
Thanks @JaredBurck I think I knew that was coming but didn't have a timeline.
It looks as though the Docker GH action that we use supports auth for pulling and thus it should be possible to use `registry.redhat.io`.
@etsauer @sabre1041 thoughts on changing registry?